docs/user/InstructionsForAutoTuner.md

```diff
@@ -106,12 +105,16 @@ For Global Routing (FastRoute) you can use:
 
 `distributed.py` scripts handles sweeping and tuning of ORFS parameters.
 
-In the sweeping mode, we manually specify the parameters search space
-in the `json`; whereas in the tuning mode - we will use Ray's Tune 
-feature to automatically optimise hyperparameters using one of the 
-algorithms listed above with autotuner `json` file. These two files 
-are of very different format, and you may refer to the examples below
-for more information. 
+The `distributed.py` script supports two modes of operation: `sweep`,
```

Comment:
```suggestion
The `distributed.py` script leverages Ray's job scheduling and management to fully utilize available hardware, albeit a single local machine or a cloud environment with thousands of CPUs. The two modes of operation are`sweep`,
```

Reply:
Remove line 106 and include the change above

Reply:
Clarify sentence to: The `distributed.py` script uses Ray's job scheduling and management to fully utilize available hardware resources from a single server to a distributed server configuration, on-premise or over the cloud with multiple CPUs.